components/molecule/stepper/src/Step/settings.js

```diff
@@ -26,6 +26,13 @@ export const getIcon = ({
 
 export const getLabel = ({label, steps, step, design, current}) => {
   if (design === DESIGN.DEFAULT) return label
-  else if (design === DESIGN.BASIC) return label
-  else if (design === DESIGN.COMPRESSED && current) return `${step}/${steps}${label ? `: ${label}` : ''}`
+  if (design === DESIGN.BASIC) return label
+  if (design === DESIGN.COMPRESSED && current)
+    return (
+      <>
+        {`${step}/${steps}`}
+        {label && `: `}
+        {label}
+      </>
```

Comment:
A question, is it necessary to return the jsx? I mean, before `getLabel` returned `string`, and with this iteration, it  would additionally return a jsx.

Reply:
should be alright... `getLabel` is also within JSX...

```
        <div {...stuff}>
          {getLabel({steps, step, design, label, current})}
        </div>
```

Reply:
https://github.com/SUI-Components/sui-components/pull/2695#issuecomment-1930031816